src/main/java/org/kairosdb/datastore/cassandra/CassandraClientImpl.java

```diff
@@ -105,20 +101,20 @@ private void logConnectionPoolConfig(PoolingOptions poolOpts) {
         }
     }
 
-    private void logConnectionStats(Cluster.Builder builder, PoolingOptions poolOpts) {
-        final Session session = m_cluster.newSession();
+	private void logConnectionStats(final Session session) {
         final ScheduledExecutorService scheduled = Executors.newScheduledThreadPool(1);
         scheduled.scheduleAtFixedRate(() -> {
             final Session.State state = session.getState();
-            for (Host host : state.getConnectedHosts()) {
-                final HostDistance distance = builder.getConfiguration().getPolicies().getLoadBalancingPolicy().distance(host);
+			final Configuration configuration = session.getCluster().getConfiguration();
+			final PoolingOptions poolingOptions = configuration.getPoolingOptions();
```

Comment:
In this PR: https://github.com/zalando-zmon/kairosdb/pull/97 you are checking is `PoolingOptions` null. Did you miss it here, or it was just an extra precaution?

Reply:
Not `PoolingOptions`, but `ProtocolOptions`.

Reply:
Oh, sorry, my bad :(